bin/pull-cftlib-images.sh

```diff
@@ -15,7 +15,10 @@ docker pull docker.elastic.co/elasticsearch/elasticsearch:7.11.1
 # Download logstash
 docker pull docker.elastic.co/logstash/logstash:7.16.1
 # Download XUI Manage Cases
-docker pull hmctspublic.azurecr.io/xui/webapp:latest
+# docker pull hmctspublic.azurecr.io/xui/webapp:latest
+# Download XUI Manage Cases - Without Work Allocation
+docker pull tdmehmet/xui-webapp:withoutWA
+docker tag tdmehmet/xui-webapp:withoutWA hmctspublic.azurecr.io/xui/webapp:latest
```

Comment:
just for reference, we shouldn't need this workaround. If you're having issues with the latest xui image, happy to help out